go/libraries/doltcore/sqle/tables.go

```diff
@@ -267,12 +272,46 @@ func (t *DoltTable) getRoot(ctx *sql.Context) (*doltdb.RootValue, error) {
 
 // GetIndexes implements sql.IndexedTable
 func (t *DoltTable) GetIndexes(ctx *sql.Context) ([]sql.Index, error) {
+	key, ok, err := t.DataCacheKey(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	if !ok {
+		tbl, err := t.DoltTable(ctx)
+		if err != nil {
+			return nil, err
+		}
+		return index.DoltIndexesFromTable(ctx, t.db.Name(), t.tableName, tbl)
```

Comment:
we don't populate the cache here, but we also never his this block

Reply:
Adding a comment to clarify